samples/crypto/ecjpake/README.rst

```diff
@@ -0,0 +1,41 @@
+.. _crypto_ecjpake:
+
+Crypto: EC J-PAKE
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The EC J-PAKE sample demonstrates how to do password-authenticated key exchange.
```

Comment:
```suggestion
The EC J-PAKE sample demonstrates how to do password-authenticated key exchange using the elliptic curve (EC) version of the password-authenticated key exchange by juggling (J-PAKE) protocol.
```

Reply:
BTW, not sure if protocol is ok to use here. Or if we should use algorithm instead (as we use in the driver config table).